backend/src/credentials/ctypeRequest.ts

```diff
@@ -0,0 +1,32 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+import { CTYPE_HASH, REQUIRED_PROPERTIES, TRUSTED_ATTESTERS } from '../config'
+
+// Here you can set which type of credential (cType) your dApp will request users to login.
+// You can change it by importing a different one.
+// The default is the Email CType by SocialKYC and SocialKYC as the Issuer
+// Establish which cTypes our dApp accepts and which attesters we trust:
+
+const trustedAttestersValues = TRUSTED_ATTESTERS.split(',') as Kilt.DidUri[]
+const requiredPropertiesValues = REQUIRED_PROPERTIES.split(',')
+
+const requiredProperties = requiredPropertiesValues.map(
```

Comment:
If you're just mapping to the same thing, the map is not required. Do you want to apply some transformations to the env string or what?

Reply:
Fixed at [99ad5a6](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/99ad5a6471d7ebe93ce3eb34c9da8f3e919cc7b3)